JustSaying.Sample.Restaurant.Models/JustSaying.Sample.Restaurant.Models.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
For models can target `netstandard2.0` instead.

Reply:
fixed